dips/dip-1.mdx

```diff
@@ -175,17 +110,19 @@ All requests between VASPs are structured as `CommandRequestObject`s.
 }
 ```
 
-### CommandResponseObject
+## CommandResponseObject
+
 All responses to a CommandRequestObject are in the form of a CommandResponseObject
 
 | Field 	     | Type 	| Required? 	| Description 	|
 |-------	     |------	|-----------	|-------------	|
 | _ObjectType    | str      | Y             | The fixed string `CommandResponseObject`. |
 | status         | str      | Y             | Either `success` or `failure`. |
-| error          | [OffChainErrorObject](#offchainerrorobject) | N | Details of the error when status == "failure" |
-| cid            | str      | N | The Command identifier to which this is a response. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included and should match the 'cid' of the CommandRequestObject. |
-
+| error          | [OffChainErrorObject](#offchainerrorobject) | N | Details of the error when status == "failure". |
+| result         | Object | N | An optional JSON object that may be defined when status == "success". |
+| cid            | str      | N | The Command identifier to which this is a response. Must be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included and must match the 'cid' of the CommandRequestObject. This field must be set unless the request to which this is responding is unparseable. |
```

Comment:
Other cases:
1. No cid in the request 
2. cid in the request is not a valid uuid
3. Blob is valid json, but there is a field type does not match (for example: command_type value is a int), causes the deserialization failure. You probably can still parse out the cid by not deserializing json blob in this case, but need call out and handle it probably.

Reply:
I think all of these imply that the request is unparseable

Reply:
What you mean unparsable? The request json string is parsed successfully for all above cases.